### DISCUSSION TITLE
A big thank you

### DISCUSSION THREAD

**Discussion by harleigh** [codewars/codewars.com#2309] [Kind words]
I wanted to send a great big thank you to you all and to let you know I am grateful for all your efforts making and maintaining the Codewars site.  You guys, via your site, make a big positive difference.  Personal example: I had to undergo, this summer, a major spinal operation that did not let me bend/twist/reach/arch for three months, and six months in I am still on the recovery road. During this time of recovery, I have been able to joyfully stay mentally active by exercising my mind through Katas, and I fell back in enjoyment of coding for fun to boot.  The site is fantastic: Very cool system of being able to gain honor, raise kyu level, see other coder's approaches; it's a real treat. Being able to have such a wide set of problems to solve, and to receive such quick feedback when solving a Kata (e.g. pass/fail submit) makes your site a very practical and applied "Learn By Doing" (a Cal Poly SLO motto) approach to learning a language (and to start the road to be a computer scientist or software engineer). I hope one day schools start to use it in their classes. I bet maintaining and improving Codewars is a job and a half, so thank you very much for all of your efforts.  Happy coding and learning to you all!

**Comment by ghost** [comment]
For what it's worth, I've undergone a short (8 months) state-sponsored webdev training course that used Codewars to get us started using Javascript as a first programming language. I agree with you that Codewars is great to learn by doing.

**Reply by Bumble-sakh** [reply]
I agree. I also use CodeWars for learning.

**Comment by wmckeever** [comment]
I'm reading this over a year later, and I hope that you have both recovered well and have been able to continue your forward momentum!

**Comment by crisperi** [comment]
Nice work code Wars
